### PR TITLE
[201739] Implement whitebox_test_secret/0

### PR DESCRIPTION
**Pull request by aYukiSekiguchi**
https://acsmine.tok.access-company.com/redmine/issues/201739

### CODE REVIEW

File: test/lib/test/config_test.exs [hunk 1]
```diff
@@ -3,6 +3,17 @@
 defmodule Antikythera.Test.ConfigTest do
   use Croma.TestCase
 
+  test "whitebox_test_secret/0 should read WHITEBOX_TEST_SECRET_JSON environment variable" do
+    env_var_name = "WHITEBOX_TEST_SECRET_JSON"
+    original = System.get_env(env_var_name)
+    System.put_env(env_var_name, ~S({"foo": "bar"}))
+    assert Antikythera.Test.Config.whitebox_test_secret()["foo"] == "bar"
```
Comment (irisTa56) [inline test/lib/test/config_test.exs:10]:
```suggestion
    assert Config.whitebox_test_secret()["foo"] == "bar"
```

**Comment by ttokit** [comment]
Can we commonize duplicate processes by introducing a private function?
For example, as in the following:
```ex
defp assert_test_secret(env_var_name, fun) do
  original = System.get_env(env_var_name)
  System.put_env(env_var_name, ~S({"foo": "bar"}))
  assert fun.()["foo"] == "bar"
  case original do
    nil -> System.delete_env(env_var_name)
    v   -> System.put_env(env_var_name, v)
  end
end

test "whitebox_test_secret/0 should read WHITEBOX_TEST_SECRET_JSON environment variable" do
  assert_test_secret("WHITEBOX_TEST_SECRET_JSON", &Config.whitebox_test_secret/0)
end

test "blackbox_test_secret/0 should read BLACKBOX_TEST_SECRET_JSON environment variable" do
  assert_test_secret("BLACKBOX_TEST_SECRET_JSON", &Config.blackbox_test_secret/0)
end
```

File: test/lib/test/config_test.exs (hunk 1 above)
Comment (aYukiSekiguchi) [reply test/lib/test/config_test.exs]: Fixed at https://github.com/access-company/antikythera/pull/122/commits/b29842edc3645e1957cc1a994d0deba09a6e2873

**Comment by aYukiSekiguchi** [comment]
> Can we commonize duplicate processes by introducing a private function?

Thank you.
I think that `secret_getter` is better than just `func`.
Fixed at https://github.com/access-company/antikythera/pull/122/commits/b29842edc3645e1957cc1a994d0deba09a6e2873

**Pull request merged**